Makefile

```diff
@@ -145,7 +145,8 @@ build-ccm-image: buildx-setup docker-pull-prerequisites ## Build controller-mana
 		--build-arg ARCH="$(ARCH)" \
 		--build-arg VERSION="$(VERSION)" \
 		--file Dockerfile \
-		--tag $(CONTROLLER_MANAGER_IMAGE) .
+		--tag $(CONTROLLER_MANAGER_IMAGE) . \
+		--provenance=false
```

Comment:
similarly to provenance, could we set `--sbom=false` as well in case the default values change later?

Reply:
Sure. Updated

Reply:
so this workaround works?

Reply:
We need to test in kubekins-e2e image so if this change does no harm then we can verify it in the pipeline.

Reply:
ok, let's merge first.